src/System/Wlog/LoggerConfig.hs

```diff
@@ -256,12 +268,14 @@ topLevelParams =
 
 instance FromJSON LoggerConfig where
     parseJSON = withObject "rotation params" $ \o -> do
-        _lcRotation      <-         o .:? "rotation"
-        _lcTermSeverity  <-         o .:? "termSeverity"
-        _lcShowTime      <- Any <$> o .:? "showTime"    .!= False
-        _lcShowTid       <- Any <$> o .:? "showTid"     .!= False
-        _lcFilePrefix    <-         o .:? "filePrefix"
-        _lcTree          <- parseJSON $ Object $ filterObject topLevelParams o
+        _lcRotation        <-                         o .:? "rotation"
+        -- TODO: Support word "All" which will include all severities at once
+        _lcTermSeverityOut <- (fmap Set.fromList) <$> o .:? "termSeveritiesOut"
```

Comment:
We have [`<<$>>`](http://hackage.haskell.org/package/universum-0.7.1.1/docs/Functor.html#v:-60--60--36--62--62-) in `unversum` to fight with nested functors.